AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -143,22 +143,12 @@ - (void)setFrame:(CGRect)rect
   ASDisplayNodeAssert(CATransform3DIsIdentity(self.transform), @"Must be an identity transform");
 #endif
 
-  BOOL useLayer = (_layer && ASDisplayNodeThreadIsMain());
-  
-  CGPoint origin      = (useLayer ? _layer.bounds.origin : self.bounds.origin);
-  CGPoint anchorPoint = (useLayer ? _layer.anchorPoint   : self.anchorPoint);
```

Comment:
I've taken a closer look and remember that I made this change after detailed profiling of the Paper app.  setFrame: is a very hot method and additional method calls actually do matter on the main thread, certainly for the iPhone 4S.  I'd like to revisit this nice clean-up once more of Pinterest is running with ASDK and I have a Paper-like target to profile these differences.

Can you revert the changes in this file, and we will land the other file?